src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -0,0 +1,627 @@
+    ca-base /etc/ssl/certs
+    crt-base /etc/ssl/private
+
+    # Default ciphers to use on SSL-enabled listening sockets.
+    # For more information, see ciphers(1SSL). This list is from:
+    #  https://hynek.me/articles/hardening-your-web-servers-ssl-ciphers/
+    ssl-default-bind-ciphers ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5:!DSS
+    ssl-default-bind-options no-sslv3
+
+defaults
+    log global
+    mode    http
+    option  httplog
+    option  dontlognull
+        timeout connect 5000
+        timeout client  50000
+        timeout server  50000
+    errorfile 400 /etc/haproxy/errors/400.http
+    errorfile 403 /etc/haproxy/errors/403.http
+    errorfile 408 /etc/haproxy/errors/408.http
+    errorfile 500 /etc/haproxy/errors/500.http
+    errorfile 502 /etc/haproxy/errors/502.http
+    errorfile 503 /etc/haproxy/errors/503.http
+    errorfile 504 /etc/haproxy/errors/504.http
+
+{{#eachAcls bundles defaultHttpPort=9000}}
+
+  {{#ifAcl 'conductr-kibana' '1' 'kibana'}}
+# ConductR - Kibana Bundle HAProxy Configuration
+frontend kibana_frontend
+  bind {{haproxyHost}}:5601
+  mode http
+  acl kibana_context_root path_beg /
+  use_backend kibana_backend if kibana_context_root
+
+backend kibana_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+
+  {{#ifAcl 'visualizer' '1.1' 'visualizer'}}
+# ConductR - Visualizer Bundle HAProxy Configuration
+frontend visualizer_frontend
+  bind {{haproxyHost}}:9999
+  mode http
+  acl visualizer_context_root path_beg /
+  use_backend visualizer_backend if visualizer_context_root
+
+backend visualizer_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+
+{{/eachAcls}}
+
+
+{{#haproxyConf bundles}}
+{{serviceFrontends}}
+{{#unless (serviceFrontends)}}
+frontend dummy
+  bind 127.0.0.1:65535
+{{/unless}}
+{{serviceBackends}}
+{{/haproxyConf}}
+```
+
+The default configuration above is written in [Handlebars](http://handlebarsjs.com/) template, and thus the template makes use of variables and helper functions made available to the template.
+
+### Dynamic proxy configuration DSL
+
+The variables and helper functions exposed to the template forms the DSL required to generate HAProxy configuration from the information available within ConductR cluster.
+
+The following variables are available to the template.
+
+* [bundles](#bundles)
+* [overrideConfigDir](#overrideConfigDir)
+
+The following helper functions are available to the template.
+
+* [eachAcls](#eachAcls)
+* [ifAcl](#ifAcl)
+* [eachBackendServer](#eachBackendServer)
+  * [serverName](#serverName)
+  * [host](#host)
+  * [port](#port)
+* [haproxyHost](#haproxyHost)
+* [serviceFrontends](#serviceFrontends)
+* [serviceBackends](#serviceBackends)
+
+
+##### haproxyHost
```

Comment:
Use `####` (h4) `h4` instead of `#####` (h5). The parent headline is a h3 headline. Apply this to all sub sections in `Dynamic proxy configuration DSL`.

Also, with the current css theme, h5 headline using the same font style as normal paragraphs (p). So we should try to avoid h5s in general.